README.md

```diff
@@ -138,10 +150,40 @@ Then: `docker-compose up`
 - `k-app-back`: which run the node process of the backend
 - `k-app-front`: which run the process of the frontend 
 
----
+It is highly recommended to run the containers in detach mode (ie. in the background) to avoid to many logs in the same terminal instance. To do it, use `docker-compose up -d`.
 
-## Testing (back only)
+You will be able to access to the logs using:
+- Backend: `docker-compose -f logs back`
+- Frontend: `docker-compose -f logs front`
 
-Launch the tests: `yarn test`
+### Notes
+
+### Environment variables
+
+Environment variables are parsed with [nconf](https://github.com/indexzero/nconf/).
+The separator used is `__` and words are transformed into camelCase.
+
+E.g.: `WEB__JWT_SECRET` will be access with `conf.get('web:jwtSecret')`.
 
-Create coverage report: `yarn run coverage`
+#### CLI
+
+The application comes with a small _cli_ which provide multiple actions. 
+Some of them are used only in production, others are used only in development.
+For example, the _cli_ is used to create the first super admin account.
+
+#### Watch lint errors
+
+As you have to follow *eslint* and *tslint* configured guidelines, 
+you can install these plugins to watch linter errors.
+
+* For [VSCode](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+* For [Webstorm](http://plugins.jetbrains.com/plugin/7494)
+
+#### Hot reload
+
+- `Backend`
+The app uses [nodemon](https://nodemon.io/) to watch for code change.
+The app will restart or reload when you edit the code.
+- `Frontend`
+The app uses `angular-cli` to watch for code change.
+The app will restart or reload when you edit the code.
```

Comment:
Should we add how to verify the email when creating acount on the App on dev mode ?